lib/zmq-kernel.coffee

```diff
@@ -306,30 +329,27 @@ class ZMQKernel extends Kernel
 
 
     destroy: ->
```

Comment:
OK I see, it makes sense.
I was just wondering because `@shutdown` has an restart option which isn't used anywhere. But this can be left for an future PR.